e2e-tests/docker_entrypoint.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-aleph-e2e-client --node "$NODE_URL" --test-cases "$TEST_CASE"
+# If test case params are not empty, run client with them. Otherwise, run without params.
+if [[ -n "${RESERVED_SEATS}" && -n "${NON_RESERVED_SEATS}" ]]; then
+  aleph-e2e-client --node "${NODE_URL}" --test-cases "${TEST_CASES}" --reserved-seats "${RESERVED_SEATS}" \
+    --non-reserved-seats "${NON_RESERVED_SEATS}"
+else
+  aleph-e2e-client --node "${NODE_URL}" --test-cases "${TEST_CASES}"
```

Comment:
if possible refactor to 
```bash
args=--node...
if [[ -n "${RESERVED_SEATS}" && -n "${NON_RESERVED_SEATS}" ]]; then
args="$args --reserved-seats=..."
fi

aleph-e2e-client $args

```

Reply:
Based on our conversation, I scrapped this in favor of an array.